projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-submit/case-edit-submit.html

```diff
@@ -51,13 +51,13 @@ <h2 class="heading-h2">{{pageTitle}}</h2>
               <ng-container *ngIf="canShowFieldInCYA(field)">
                 <tr ccdLabelSubstitutor [caseField]="field" [hidden]="field.hidden"
                     [formGroup]="editForm.controls['data']" [contextFields]="contextFields">
-                  <th *ngIf="!isLabel(field) && !isCaseFlagSubmission" class="valign-top case-field-label"><span class="text-16">{{field.label}}</span></th>
+                  <th *ngIf="!isLabel(field) && !caseFlagField" class="valign-top case-field-label"><span class="text-16">{{field.label}}</span></th>
                   <td class="form-cell case-field-content" [attr.colspan]="isLabel(field) ? '2' : '1'">
                     <ccd-field-read
                       [formGroup]="editForm.controls['data']" [topLevelFormGroup]="editForm.controls['data']"
                       [caseField]="summaryCaseField(field)" [context]="paletteContext" [caseFields]="contextFields"></ccd-field-read>
                   </td>
-                  <ng-container *ngIf="eventTrigger.id !== 'createCaseFlag'">
+                  <ng-container *ngIf="eventTrigger.id !== 'createCaseFlag' && eventTrigger.id !== 'requestSupport'">
```

Comment:
@MateiHMCTS - this won't work if an HMCTS Service configures the event trigger ID to be something other than `createCaseFlag` or `requestSupport`. These values have come from the Definition file. The event trigger ID can be extracted from the URL. For example, in this URL: https://manage-case.aat.platform.hmcts.net/cases/case-details/1673960732219034/trigger/requestSupport/requestSupportRequestSupportFormPage, the trigger ID occurs after `trigger/` and before the last part of the path, i.e. `requestSupport`. Likewise, in this URL: https://manage-case.aat.platform.hmcts.net/cases/case-details/1673960732219034/trigger/updateCase/updateCaseSingleFormPage, the trigger ID is `updateCase`.